index.html

```diff
@@ -250,13 +357,23 @@ <h3>Can You Recognise Your Favourite Singer?</h3>
       <div class="heading">Zestful Dance</div>
       <div class="container">
         <div class="content">
-          <h1>Lets Dance</h1>
-          <span class="girl">💃</span>
+          <div class="flip-card">
+            <div class="flip-card-inner">
+              <div class="flip-card-front">
+                <h1>Lets Dance</h1>
+                
+               </div>
+              <div class="flip-card-back">
+                <span><i class="bx bx-tada bxs-medal"></i></span>
+                <h3>Test your memory by this music game</h3>
```

Comment:
You have mistakenly added this in the dance section. Please fix

Reply:
@nilisha-jais sorry for the mistake.Its rectified now.